app/controllers/posts_controller.rb

```diff
@@ -185,6 +185,12 @@ def delete_post
   end
 
   def cast_spam_flag
+    feedback = Feedback.new(feedback_type: 'tpu-',
+                            user_id: current_user.id,
+                            post_id: @post.id)
+
+    feedback.save || flash[:danger] = 'Unable to save feedback. Ping Undo.'
```

Comment:
Prefer the original unless here, assign in a conditional feels icky

Reply:
@ArtOfCode- Yes I prefer the original unless too but rubocop doesn't think so.

Reply:
@ArtOfCode- Yes I prefer the original unless too but rubocop doesn't think so.

Reply:
I think rubocop would be equally happy with `flash... unless feedback.save`.

Reply:
@thesecretmaster I was thinking that it was too unintuitive (I did think of it, though) and was hesitant to write code in that way. Anyway I've done this as of now.